### DISCUSSION TITLE
About the fp run in dpgen

### DISCUSSION THREAD

**Discussion by Niuyutao** [deepmodeling/dpgen#1529] [Q&A]
Running to the fp stage of DPGEN, it has been running for more than 10 hours, and it has been generating some .sub and .sub.run files in the temporary files, and I don't see any traces of computation in the task.000.0000 and other files, has anyone encountered this situation? I am using supercomputing dpgen, do you feel that it is the reason for calling fp? I am looking forward to your reply.
Here are the contents of the document.
![0c0da2e7022cac57a2ce6acef7f1fad](https://github.com/deepmodeling/dpgen/assets/167728606/8c1f2646-abdf-4063-8287-14821e184b29)
![1713752481596](https://github.com/deepmodeling/dpgen/assets/167728606/cdd11688-0186-4c45-9070-1bddfa8aa073)

**Comment by njzjz** [comment]
If `..._flag_if_job_task_fail` was not generated, your Slurm didn't run it.
